diff -Naur i8kutils-1.25.old/debian/i8kutils.i8kmon i8kutils-1.25/debian/i8kutils.i8kmon
--- i8kutils-1.25.old/debian/i8kutils.i8kmon	2003-12-29 16:54:27.000000000 +0200
+++ i8kutils-1.25/debian/i8kutils.i8kmon	2007-08-06 04:06:14.000000000 +0300
@@ -15,7 +15,7 @@
 PROC_I8K=/proc/i8k
 NAME=i8kmon
 DESC="Dell Inspiron fan/cpu-temperature monitor"
-I8KMON_ARGS="--daemon"
+I8KMON_ARGS="--daemon --nouserconfig"
 
 set -e
 
diff -Naur i8kutils-1.25.old/i8kmon i8kutils-1.25/i8kmon
--- i8kutils-1.25.old/i8kmon	2004-01-05 13:12:52.000000000 +0200
+++ i8kutils-1.25/i8kmon	2007-08-06 04:17:24.000000000 +0300
@@ -34,6 +34,7 @@
     proc_ac26	/proc/acpi/ac_adapter/AC/state
     i8kfan	/usr/bin/i8kfan
     geometry	{}
+    use_conf	1
     auto	1
     daemon	0
     verbose	0
@@ -87,7 +88,7 @@
 	3	{{2 2}  70 128  75 128}
     }
 
-    foreach file [list $config(sysconfig) $config(userconfig)] {
+    foreach file [list $config(sysconfig)] {
 	if {[file exists $file]} {
 	    if {$config(verbose) != 0} {
 		puts "# reading $file"
@@ -95,6 +96,16 @@
 	    source $file
 	}
     }
+    if {$config(use_conf) != 0} {
+	foreach file [list $config(userconfig)] {
+	    if {[file exists $file]} {
+		if {$config(verbose) != 0} {
+		    puts "# reading $file"
+		}
+		source $file
+	    }
+	}
+    }
 
     set t_high 0
     foreach key {0 1 2 3} {
@@ -655,6 +666,7 @@
 
     -d|--daemon			run in daemon mode without user interface
     -n|--noauto			don't control automatically the fans
+   -nc|--nouserconfig		don't use \$HOME/.i8kmon
     -v|--verbose		report status on stdout
     -g|--geometry <geometry>	set applet geometry
     -t|--timeout <seconds>	set poll timeout
@@ -687,6 +699,12 @@
 	    --noauto - -na - -n {
 		set config(auto) 0
 	    }
+	    --userconfig - -c {
+	    	set config(use_conf) 1
+	    }
+	    --nouserconfig - -nc {
+	    	set config(use_conf) 0
+	    }
 	    --verbose - -v {
 		set config(verbose) 1
 	    }
diff -Naur i8kutils-1.25.old/i8kmon.1 i8kutils-1.25/i8kmon.1
--- i8kutils-1.25.old/i8kmon.1	2002-08-22 19:20:33.000000000 +0300
+++ i8kutils-1.25/i8kmon.1	2007-08-06 04:10:12.000000000 +0300
@@ -66,6 +66,13 @@
 Monitor cpu temperature and fan status but don't control
 automatically the fans.
 .TP 
+\fB\-c\fR, \fB\-\-userconfig\fR
+Attempt to source the ~/.i8kmon configfile in addition to /etc/i8kmon.
+This is the default setting.
+.TP 
+\fB\-nc\fR, \fB\-\-nouserconfig\fR
+Don't attempt to source the ~/.i8kmon configfile, only /etc/i8kmon.
+.TP 
 \fB\-v\fR, \fB\-\-verbose\fR
 Report hardware status and program actions on stdout.
 .TP 
@@ -122,7 +129,8 @@
 the line "set config(daemon) 1". Note the the /etc/i8kmon configfile is not
 installed by the i8kutils package because the program is designed to be run
 by normal users. If you want to use it as daemon you must create the config
-file yourself.
+file yourself. In this case, the \fB\-\-nouserconfig\fR option can sometimes
+also help by limiting it to sourcing /etc/i8kmon (and not ~/.i8kmon).
 .SH "FILES"
 .LP 
 \fI/proc/i8k\fP
